flight/targets/flyingf4/board-info/pios_board.h

```diff
@@ -223,7 +223,27 @@ extern uintptr_t pios_com_debug_id;
 //-------------------------
 #define PIOS_ADC_SUB_DRIVER_MAX_INSTANCES       3
 
-#define VREF_PLUS				3.0f
+// PIOS_ADC_PinGet(0) = PC_11
+// PIOS_ADC_PinGet(1) = PC_12
+//-------------------------
+#define PIOS_DMA_PIN_CONFIG                                                                   \
```

Comment:
@mlyle  I'm not understand. 
Is needed reformat code? 
What better way to do it?